src/utils/assignParents.ts

```diff
@@ -0,0 +1,22 @@
+import { Person } from '../types/Person';
+
+export const assignParents = (people: Person[]) => {
+  const peopleWithParents: Person[] = people.map(person => {
+    const personWithParents = { ...person };
```

Comment:
Instead of creating a new object `personWithParents`, you can directly add the `mother` and `father` properties to the `person` object while returning it in the `map` function.